.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Contribution			
+
+New contributions are more than welcome! ❤️ 
+
+## Contribution workflow
+
+Whether you are planning to add
+a new dataset, add data to an existing one, or simply fix a typo or a bug
+wherever in this repository, you will have to follow the following workflow:
+
```

Comment:
Would re-write:

Contributors should perform the workflow below if:
1. you are planning to add a new dataset
2. you are adding data to an already existing dataset
3. or make any changes (e.g. fixing a typo or a bug)